deps/poolboy-1.5.1/Makefile

```diff
@@ -0,0 +1,34 @@
+REBAR = $(shell command -v rebar || echo ./rebar)
```

Comment:
@fenollp rebar came with the repo, i'll remove it if you update our rebar in utils which is quite old :)